pkg/annotations/ingress_annotations.go

```diff
@@ -60,6 +61,15 @@ func IsApplicationGatewayIngress(ing *v1beta1.Ingress) (bool, error) {
 	return controllerName == ApplicationGatewayIngressClass, err
 }
 
+// IsIstioGatewayIngress checks if this gateway should be handled by AGIC or not
+func IsIstioGatewayIngress(gateway *v1alpha3.Gateway) (bool, error) {
+	val, ok := gateway.Annotations[IstioGatewayKey]
```

Comment:
Since both `val` and `ok` are used in the scope of the conditional below -  we could combine the two lines -- `if val, ok := gateway.Annotations[IstioGatewayKey]; ok {`